.gemini/styleguide.md

```
@@ -0,0 +1,107 @@
+- Meta: `[tests]`, `[ci]`, `[docs]`, `[maintenance]`
+- Package-specific: any `[<package-name>]` matching a directory under `packages/`
+
+Each commit must have a `Signed-off-by:` trailer (produced by `git commit --signoff`).
+
+PR body must contain a release note block:
+
+```text
+ ```release-note
+ [component] Human-readable changelog entry
+ ```
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)